causing/graph.py

```diff
@@ -231,9 +231,17 @@ def specific_color_str(wei: float, line_colored: bool) -> str:
         show_in_percent,
         node_name,
     )
-    dot_str = "digraph { \n" + form + x_dot + y_dot + "        }"
 
-    dot_str = graphviz.unflatten(dot_str, stagger=3)
+    if invisible_node:
+        invis_node = (
+            '         "{}" -> "{}" [style = "invisible", arrowhead="none"];\n'.format(
+                invisible_node[0][0], invisible_node[0][1]
```

Comment:
This looks like an invisible edge, not an invisible node.

It also looks like it is a list of edges (otherwise we would expect just `invisible_node[0], invisible_node[1]`), but we only use the first element from that list. Allowing multiple edges to be passed is reasonable, we then we should
* use a plural form of the variable name
* create dot output for all edges, not just the first